documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,45 +1,86 @@
 
 === Screen Orientation
 
-LCD displays have an optimum viewing angle, and depending on how the screen is mounted it may be necessary to change the orientation of the display to give the best results. By default, the Raspberry Pi Touch Display and Raspberry Pi are set up to work best when viewed from slightly above, for example on a desktop. If viewing from below, you can physically rotate the display, and then tell the system software to compensate by running the screen upside down.
+LCD displays have an ideal viewing angle, and it may be necessary to change the orientation of the display to find the perfect angle. If you need to physically rotate the display, or mount it in a specific position, you can use software to adjust the orientation of the screen to better match your setup.
 
-To set screen orientation when running the graphical desktop, select the `Screen Configuration` option from the `Preferences` menu. Right click on the DSI display rectangle in the layout editor, select Orientation then the required option.
+To set the screen orientation from the desktop environment, select the **Screen Configuration** option from the **Preferences** menu. Right click on the DSI display rectangle in the layout editor, select **Orientation**, then pick the best option to fit your needs.
 
-To set screen orientation when in console mode, you will need to edit the kernel command line to pass the required orientation to the system.
+To set screen orientation on a headless setup, or while in console mode, you need to edit the kernel's `cmdline.txt` file to pass the required orientation to the system.
 
 [,bash]
 ----
 sudo nano /boot/firmware/cmdline.txt
 ----
 
-To rotate by 90 degrees clockwise, add the following to the cmdline, making sure everything is on the same line, do not add any carriage returns. Possible rotation values are 0, 90, 180 and 270.
+To rotate the screen, add the following to the cmdline. Make sure everything is on the same line; do not add any carriage returns. Possible rotation values are 0, 90, 180 and 270.
 
 ----
 video=DSI-1:800x480@60,rotate=90
 ----
 
-NOTE: In console mode it is not possible to rotate the DSI display separately to the HDMI display, so if you have both attached they must both be set to the same rotation value.
+NOTE: In console mode it is not possible to rotate the DSI display separately from the HDMI display. If you have both attached they need to be set to the same rotation value.
 
 === Touchscreen Orientation
 
-Additionally, you have the option to change the rotation of the touchscreen independently of the display itself by adding a `dtoverlay` instruction in `config.txt`, for example:
+You also have the option to change the rotation of the active touchscreen area independently of the display itself by adding a `dtoverlay` instruction in `config.txt`, as in the following example:
 
 `dtoverlay=rpi-ft5406,touchscreen-swapped-x-y=1,touchscreen-inverted-x=1`
```

Comment:
Why is this in backticks, not a source block?

Reply:
Still getting used to the various bits and bobs in adocs. Fixed.